docs/sources/operators-guide/running-production-environment-with-helm/configuration-with-helm.md

```diff
@@ -0,0 +1,531 @@
+---
+title: "Manage the configuration of Grafana Mimir with Helm"
+menuTitle: "Manage the configuration of Grafana Mimir with Helm"
+description: "Learn how to customize, secure and update the Grafana Mimir configuration using the Helm chart."
+weight: 80
+---
+
+# Manage the configuration of Grafana Mimir with Helm
+
+The `mimir-distributed` Helm chart provides interfaces to set Grafana Mimir [configuration parameters]({{< relref "../configure/reference-configuration-parameters/" >}}) and customize how Grafana Mimir is deployed on a Kubernetes cluster. This document is about the configuration parameters.
+
+## Overview
+
+The Grafana Mimir configuration can be managed through the Helm chart or supplied via a user managed object.
```

Comment:
Added a hyphen since "user-managed" functions as an adjective for object. 
```suggestion
The Grafana Mimir configuration can be managed through the Helm chart or supplied via a user-managed object.
```